### PR TITLE
Configuration: re-locate landscape code snippets

### PR DESCRIPTION
**Pull request by rachelcarmena**
Reason: `arrow-site` repository will be moved to a just Jekyll content repository without documentation validation because landscape code snippets have been located into their corresponding repositories. In this way they will be updated according to Arrow libraries changes without waiting to the release time.

Changes:

* Remove documentation validation for `arrow-site`
* Adapt configuration.
* Remove unnecessary steps.

### CODE REVIEW

File: docs/libraries/how-to-run-the-website-in-your-local-server.md [hunk 1]
```diff
@@ -19,14 +19,9 @@ It can be useful to know an alternative in order to avoid running all the steps
 ```sh
 git clone https://github.com/arrow-kt/arrow-site.git
 cd arrow-site
-perl -pe "s/\/docs//g" -i docs/_data/features.yml
-./gradlew runAnk
+rm docs/index.md
```
Comment (nomisRev) [inline docs/libraries/how-to-run-the-website-in-your-local-server.md:22]: Why is this necessary as preparation for the website? 🤔
If I understood correctly from this update, I only need to overwrite the new changes I want to test in the `arrow-site` dir.

**Review by nomisRev — APPROVED** [review]
👏 👏 This will be much easier to test website changes!

File: docs/libraries/how-to-run-the-website-in-your-local-server.md [hunk 2]
```diff
@@ -44,21 +39,21 @@ cd <arrow-library>
 ./gradlew buildArrowDoc
 ```
 
-Then copy the result to the previous directory:
+Then copy the result to the `docs` directory:
 
 ```sh
-cp -r <arrow-library>/arrow-docs/build/site/* arrow-site/build/site/
+cp -r <arrow-library>/arrow-docs/build/site/* arrow-site/docs
```
Comment (raulraja) [inline docs/libraries/how-to-run-the-website-in-your-local-server.md:45]: Can this step be automated?

File: scripts/site-run-ank.sh [hunk 3]
```diff
@@ -1,5 +1,3 @@
 #!/bin/bash
 
-set -e
-cd $BASEDIR/arrow-site
-./gradlew runAnk
+echo "-- runAnk is not executed for the site"
```
Comment (raulraja) [inline scripts/site-run-ank.sh:3]: How do the snippets get type-checked if Ank no longer runs?

**Review by raulraja — APPROVED** [review]

File: docs/libraries/how-to-run-the-website-in-your-local-server.md (hunk 1 above)
Comment (rachelcarmena) [reply docs/libraries/how-to-run-the-website-in-your-local-server.md:22]: Right! It's removed because maybe you don't have all the landscape snippets (core, fx, optics) and it won't work when generating the index. However, they are the manual steps and the recommended way is via Gradle task: https://github.com/arrow-kt/arrow/blob/re-org-site/docs/libraries/how-to-run-the-website-in-your-local-server.md#via-gradle-task-recommended

```
./gradlew runArrowSite
```

File: docs/libraries/how-to-run-the-website-in-your-local-server.md (hunk 2 above)
Comment (rachelcarmena) [reply docs/libraries/how-to-run-the-website-in-your-local-server.md:45]: Yes, it's already automated: 

```
./gradlew runArrowSite
```

Just showing an alternative way with manual steps.

**Comment by rachelcarmena** [comment]
Thanks for the reviews @raulraja , @nomisRev !! I made a few checks before merging it after other priorities

**Comment by rachelcarmena** [comment]
Thanks for the reviews @raulraja , @nomisRev !! I'll make a few checks before merging it after other priorities
